05_syntax_lab/07.py

```diff
@@ -1,8 +1,37 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-
-After each guess print a hint "too large" or "too small" to the user.
-
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
+from random import randint
+
+print "Exercise # 7"
+
+num = randint(1, 100)
+
+mistake_ratio = randint(5, 10)
+i = int(0)
+
+print "Number is: %d " % (num)
+
+
+while True:
+    print "What is your Guess?"
+    guess = int(raw_input())
+
+    if guess == num:
+        print "Its a Hit - Well Done!!!"
+        break
+    if i < mistake_ratio: 
+        if guess > num:
+            print "Try smaller number"
+        else:
+            print "Try higher number"
+            
+    if i == mistake_ratio: 
+        if guess > num:
+            print "Try higher number"
+        else:
+            print "Try smaller number"
+        i = int(-1)
+
+    i=i+1
+    
+    
+            
+    

```

Comment:
Well Done!

I see you're very close to finishing the second lab too, so as a bonus you can try reducing some repetitions in this number guessing game. It may require concepts we haven't learned yet so feel free to explore